### ISSUE TITLE
featuredURL function needs default argument

### ISSUE THREAD

**Issue by erwstout** [factor1/prelude-wp#86]
```php
function featuredURL($size){
  $thumb = wp_get_attachment_image_src( get_post_thumbnail_id($post->ID), $size );
  $url = $thumb['0'];
  echo $url;
}
```

Should default to `full`

```php
function featuredURL($size = 'full'){
  $thumb = wp_get_attachment_image_src( get_post_thumbnail_id($post->ID), $size );
  $url = $thumb['0'];
  echo $url;
}
```

**Comment by bebaps** [comment]
As discussed, there was an instance when the `$post->ID` was not set. I don't remember if I had used this function on a particular page type (ie archive) or if I used it within a template part, but I had to modify it to pass in `null` by default (which is the default parameter accepted by the `get_post_thumbnail_id()` anyway so really I just had to restore some default functionality).

```
function get_featured_image( $post_id = null, $size = 'Full' ) {
  if ( !has_post_thumbnail( $post_id ) ) {
    return;
  }

  $image_url = wp_get_attachment_image_src( get_post_thumbnail_id( $post_id ), $size );

  return $image_url[ 0 ];
}
```

**Comment by erwstout** [comment]
Can we shelf that until we find when/where it breaks? I personally haven't ran into that issue so I'd like to know a little more about it before adding a fix.  Maybe if you run into it again open an issue describing the steps to reproduce?

**Comment by erwstout** [comment]
Fixed in `3.3.11`
